packages/v3-sdk/src/utils/tickMath.ts

```diff
@@ -80,7 +80,7 @@ export abstract class TickMath {
    * @param sqrtRatioX96 the sqrt ratio as a Q64.96 for which to compute the tick
    */
   public static getTickAtSqrtRatio(sqrtRatioX96: bigint): number {
-    invariant(sqrtRatioX96 >= TickMath.MIN_SQRT_RATIO && sqrtRatioX96 < TickMath.MAX_SQRT_RATIO, 'SQRT_RATIO')
+    invariant(sqrtRatioX96 >= TickMath.MIN_SQRT_RATIO && sqrtRatioX96 <= TickMath.MAX_SQRT_RATIO, 'SQRT_RATIO')
```

Comment:
I think we should update the price limit instead. When we set price range from 0 -> max, the sqrtRatioLimit should be [TickMath.MIN_SQRT_RATIO + 1n, TickMath.MAX_SQRT_RATIO - 1n]

Reply:
Done

Reply:
Should only adjust when meet the max or min